src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -959,7 +960,7 @@ private void RunConnectRequestHandlerTask(ConnectParams connectParams)
                     };
                     await ServiceHost.SendEvent(ConnectionCompleteNotification.Type, result);
                 }
-            });
+            }).ContinueWithOnFaulted(null);

```

Comment:
should we write these faults to the diag log?

Reply:
Yep, that's part of the method.